.eslintignore

```diff
@@ -3,6 +3,8 @@ node_modules/*
 **/.*
 **/dist/*
 *.dist.js
+*.dist.ts
```

Comment:
I can tell who lints. 

Some time, I'd love for you to review my lint setup on my vim stack.  I think I could utilize that tooling better.